Common/gui/guidefines.h

```diff
@@ -174,7 +175,8 @@ enum GUITextBoxFlags
 enum GuiSvgVersion
 {
     kGuiSvgVersion_Initial  = 0,
-    kGuiSvgVersion_350      = 1
+    kGuiSvgVersion_350      = 1,
+    kGuiSvgVersion_399      = 2
```

Comment:
Please give new save format a much higher number in case there will be change in 3.* branch